tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM dredd/base
+FROM giovannicuriel/dredd-python
```

Comment:
dojot/dredd-python

Reply:
For now, there is no such image in the docker repository As soon as dojot/dredd-python contains one image for python 3, I'll update this.

Reply:
Which will be in a few moments.